src/services/ImageRepository.cpp

```diff
@@ -241,7 +243,7 @@ static HRESULT ConvertBitmapSource(_In_ RECT rcDest, _In_ IWICBitmapSource* pOri
     return hr;
 }
 
-static HRESULT CreateDIBFromBitmapSource(_In_ IWICBitmapSource *pToRenderBitmapSource, _Out_ HBITMAP& hBitmapInOut)
+static HRESULT CreateDIBFromBitmapSource(_In_ IWICBitmapSource *pToRenderBitmapSource, _Inout_ HBITMAP& hBitmapInOut)
```

Comment:
You can't really `new` a GDI Handle, plus it already says `InOut`,
You can do something similar with `std::unique_ptr `but that'll be saved for later.